_pkgdown.yml

```diff
@@ -14,25 +16,23 @@ navbar:
           href: StandardReportSite.html
         - text: Country Report
           href: StandardReportCountry.html
-    articles:
-      text: Articles
-      menu:
-        - text: Cookbook
-          href: articles/Cookbook.html
-        - text: Step-by-Step Analysis Workflow
-          href: articles/DataAnalysis.html
-        - text: Data Model
-          href: articles/DataModel.html
-        - text: Contributor Guidelines
-          href: articles/ContributorGuidelines.html
-        - text: KRI Method
-          href: articles/KRI Method.html
-        - text: Qualification Report for the {gsm} R Package
-          href: articles/Qualification.html
-        - text: Qualification Workflow
-          href: articles/QualificationWorkflow.html
-        - text: v1.9.0 Data Model Update Reference
-          href: articles/v1_9_0-data-model-update-reference.html
+    # articles:
+    #   text: Articles
+    #   menu:
+    #     - text: Cookbook
+    #       href: articles/Cookbook.html
+    #     - text: Step-by-Step Analysis Workflow
+    #       href: articles/DataAnalysis.html
+    #     - text: Data Model
+    #       href: articles/DataModel.html
+    #     - text: Contributor Guidelines
+    #       href: articles/ContributorGuidelines.html
+    #     - text: KRI Method
+    #       href: articles/KRI Method.html
+    #     - text: Qualification Report for the {gsm} R Package
+    #       href: articles/Qualification.html
+    #     - text: Qualification Workflow
+    #       href: articles/QualificationWorkflow.html
```

Comment:
```suggestion
```

We don't want them called out here in any case (they should be their own top-level thing, like references, so pkgdown groks that they're real articles)... but we don't need it at all unless we want to organize them. If we don't mention them, pkgdown will add them all.

Reply:
ok, i'll leave it as is for now, and determine if we want to update the order to have most relevant articles up top in a different issue